src/components/SecondaryPanes/Expressions.css

```diff
@@ -66,20 +69,32 @@
 
 .expression-container {
   border: 1px;
-  padding: 0.6em 1em 0.6em 0.5em;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  padding-inline-start: 20px;
```

Comment:
You're using 19px above to account for the 1px border. We could use the same here.

Reply:
Do we? I wasn't sure about this border - it's not really displayed so it never factors into the measurement, unlike the input form. Setting this to 19px would mean we're short by 1px here.

Let me know if I'm missing anything, otherwise it might be good to take the border value out.

Reply:
You're right, let's keep the 20px and remove the `border: 1px` to clean things up a bit.

Reply:
Took this out.